### ISSUE TITLE
'diff: no' causes error

### ISSUE THREAD

**Issue by PhillWide** [ansible-community/ansible-lint#610]
# Issue Type
- Bug report

# Ansible and Ansible Lint details

```
ansible --version
ansible 2.8.3
  config file = /etc/ansible/ansible.cfg
  configured module search path = ['/root/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/local/lib/python3.7/site-packages/ansible
  executable location = /usr/local/bin/ansible
  python version = 3.7.4 (default, Jul 13 2019, 14:04:11) [GCC 8.3.0]

ansible-lint --version
ansible-lint 4.1.0
```

- ansible installation method:  OS package
- ansible-lint installation method: pip3

# Desired Behaviour

No error - it is part of ansible https://docs.ansible.com/ansible/2.5/user_guide/playbooks_checkmode.html#showing-differences-with-diff

# Actual Behaviour (Bug report only)

Traceback (most recent call last):
  File "/usr/local/bin/ansible-lint", line 10, in <module>
    sys.exit(main())
  File "/usr/local/lib/python3.7/site-packages/ansiblelint/__main__.py", line 187, in main
    matches.extend(runner.run())
  File "/usr/local/lib/python3.7/site-packages/ansiblelint/__init__.py", line 282, in run
    skip_list=self.skip_list))
  File "/usr/local/lib/python3.7/site-packages/ansiblelint/__init__.py", line 174, in run
    matches.extend(rule.matchtasks(playbookfile, text))
  File "/usr/local/lib/python3.7/site-packages/ansiblelint/__init__.py", line 86, in matchtasks
    for task in ansiblelint.utils.get_normalized_tasks(yaml, file):
  File "/usr/local/lib/python3.7/site-packages/ansiblelint/utils.py", line 541, in get_normalized_tasks
    res.append(normalize_task(task, file['path']))
  File "/usr/local/lib/python3.7/site-packages/ansiblelint/utils.py", line 474, in normalize_task
    task = normalize_task_v2(task)
  File "/usr/local/lib/python3.7/site-packages/ansiblelint/utils.py", line 374, in normalize_task_v2
    action, arguments, result['delegate_to'] = mod_arg_parser.parse()
  File "/usr/local/lib/python3.7/site-packages/ansible/parsing/mod_args.py", line 292, in parse
    module_loader.has_plugin(item, collection_list=self._collection_list):
  File "/usr/local/lib/python3.7/site-packages/ansible/plugins/loader.py", line 514, in has_plugin
    return self.find_plugin(name, collection_list=collection_list) is not None
  File "/usr/local/lib/python3.7/site-packages/ansible/plugins/loader.py", line 505, in find_plugin
    'Module "%s" shadows the name of a reserved keyword. Please rename or remove this module. Found at %s' % (name, plugin)
ansible.errors.AnsibleError: Module "diff" shadows the name of a reserved keyword. Please rename or remove this module. Found at /usr/bin/X11/diff

# source

```yaml
- name: get Sentry repository
  git:
    repo: https://github.com/getsentry/onpremise.git
    dest: /srv/sentry
    version: ea78661e360aafd436a8248e14365fe0db5e179d  # version 9.0.0 dirty
  diff: no
  tags: sentry
```
If I remove the diff: no, the lint passes

**Comment by ssbarnea** [comment]
No longer reproduces with code from master.
